docs/mine-hnt/data-only-guides/dragino-pg1301.mdx

```diff
@@ -0,0 +1,290 @@
+   Oct 02 15:00:53 raspi lora_pkt_fwd[878]: REPORT~ ### [UPSTREAM] ###
+   Oct 02 15:00:53 raspi lora_pkt_fwd[878]: REPORT~ # RF packets received by concentrator: 1
+   Oct 02 15:00:53 raspi lora_pkt_fwd[878]: REPORT~ # CRC_OK: 0.00%, CRC_FAIL: 100.00%, NO_CRC: 0.00%
+   Oct 02 15:00:53 raspi lora_pkt_fwd[878]: REPORT~ # RF packets forwarded: 0 (0 bytes)
+   Oct 02 15:00:53 raspi lora_pkt_fwd[878]: REPORT~ # PUSH_DATA datagrams sent: 0 (0 bytes)
+   Oct 02 15:00:53 raspi lora_pkt_fwd[878]: REPORT~ # PUSH_DATA acknowledged: 0.00%
+   Oct 02 15:00:53 raspi lora_pkt_fwd[878]: REPORT~ ### [DOWNSTREAM] ###
+   Oct 02 15:00:53 raspi lora_pkt_fwd[878]: REPORT~ # PULL_DATA sent: 3 (100.00% acknowledged)
+   Oct 02 15:00:53 raspi lora_pkt_fwd[878]: REPORT~ # PULL_RESP(onse) datagrams received: 0 (0 bytes)
+   Oct 02 15:00:53 raspi lora_pkt_fwd[878]: REPORT~ # RF packets sent to concentrator: 0 (0 bytes)
+   Oct 02 15:00:53 raspi lora_pkt_fwd[878]: REPORT~ # TX errors: 0
+   Oct 02 15:00:53 raspi lora_pkt_fwd[878]: REPORT~ # BEACON queued: 0
+   Oct 02 15:00:53 raspi lora_pkt_fwd[878]: REPORT~ # BEACON sent so far: 0
+   Oct 02 15:00:53 raspi lora_pkt_fwd[878]: REPORT~ # BEACON rejected: 0
+   Oct 02 15:00:53 raspi lora_pkt_fwd[878]: REPORT~ ### [PPS] ###
+   Oct 02 15:00:53 raspi lora_pkt_fwd[878]: REPORT~ # SX1301 time (PPS): 2463443
+   Oct 02 15:00:53 raspi lora_pkt_fwd[878]: RXTX~ {"stat":{"time":"2021-10-02 18:58:53 GMT","rxnb":1,"rxok":0,"rxfw":0,"ackr":0.0,"dwnb":0,"txnb":0}}
+   Oct 02 15:00:53 raspi lora_pkt_fwd[878]: REPORT~ ################## Report at: 2021-10-02 18:59:23 GMT ##################
+   Oct 02 15:00:53 raspi lora_pkt_fwd[878]: REPORT~ ### [UPSTREAM] ###
+   Oct 02 15:00:53 raspi lora_pkt_fwd[878]: REPORT~ # RF packets received by concentrator: 0
+   ```
+   It can take a minute or two for everything to start up.
+8. Enable the LoRa Packet Forwarder to start up at boot time:
+   `sudo systemctl enable lorapktfwd.service` . If the service is already
+   enabled at boot, the command will return no output.
+
+## Install the Helium Gateway Service
+
+1. Download latest helium-gateway release package from
+   https://github.com/helium/gateway-rs/releases ending in -raspi234.deb
+2. Install the package using `dpkg`. Example:
+   `sudo dpkg -i helium-gateway-v1.0.0-alpha.16-raspi234.deb` (Replace
+   helium-gateway-v1.0.0-alpha.16-raspi234.deb) with the version you downloaded.
+
+### Region configuration
+
+The default region of the _helium_gateway_ service is US915, so you can continue
+with the next section if you are using this one. Otherwise, you must set your
+region frequency in `/etc/helium_gateway/settings.toml`. Add the following line
+at the beginning of the file:
+
+```
+region = "<region>"
+```
+
+Possible values are:
+`US915 | EU868 | EU433 | CN470 | CN779 | AU915 | AS923_1 | AS923_2 | AS923_3 | AS923_4 | KR920 | IN865`.
```

Comment:
Thanks for all the fixes. This is really the last major one I'm seeing. For these lines that are pretty long, it's probably best to put it in triple single quotes instead so users can scroll horizontally instead of it wrapping to the next line.

````
```
text
```
````